land-of-wizards/core.clj

```diff
@@ -0,0 +1,57 @@
+(ns land-of-wizards.core
+  (:require [clojure.string :as str]
+            [land-of-wizards.data :refer :all]))
+
+(comment "The first command to have is a command that tells about the location the wizard is standing in. So what would a function need to describe a location in a world? Well, it would need to know the location to describe and would need to be able to look at a map and find that location on the map. Here's a function, that does exactly that")
+
+(defn describe-location [location nodes]
```

Comment:
Maybe remove the actual solutions out of the repo? We can have the empty function structure.

This is so that attendees can work off the same file. We usually get folks to get the repo on their machine so that they can copy paste and try stuff in their REPL.